src/common/Analytics/index.ts

```diff
@@ -0,0 +1,10 @@
+export {
```

Comment:
Can we just do `export * from '...'` instead? That will make this file much more maintainable.